src/experiment_prototype/experiment_slice.py

```diff
@@ -844,19 +853,17 @@ def check_freq(cls, freq, values):
                     f"freq is within a restricted frequency range {freq_range}"
                 )
 
-        # TODO review issue #195
-        tx_center = rx_center = 12000  # Default tx and rx center freq value
-        if "txctrfreq" in values:
-            tx_center = values["txctrfreq"]
-        if "rxctrfreq" in values:
-            rx_center = values["rxctrfreq"]
+        # TODO review issue #195 - Characterize transmit waveforms near edge of tx bandwidth
+        # Default center freqs to 12000 if not set in values
+        tx_center = values.get("txctrfreq", 12000)
+        rx_center = values.get("rxctrfreq", 12000)
 
         # Frequency must be withing bandwidth of rx and tx center frequency
-        if (freq > values["rx_bound_freqs"][1]) or (freq < values["rx_bound_freqs"][0]):
+        if (freq > values["rx_freq_bounds"][1]) or (freq < values["rx_freq_bounds"][0]):
             raise ValueError(
                 f"Slice frequency is outside bandwidth around rx center frequency {int(rx_center)}"
             )
-        if (freq > values["tx_bound_freqs"][1]) or (freq < values["tx_bound_freqs"][0]):
+        if (freq > values["tx_freq_bounds"][1]) or (freq < values["tx_freq_bounds"][0]):
```

Comment:
This should only fail if `rxonly` is `False`. If `rxonly` is `True`, then we don't need to care what the `tx_freq_bounds` are